pkg/op-data/op_data/db/models.py

```diff
@@ -61,12 +61,22 @@ class RawTopic(Model):
     rawData = fields.JSONField()
     lastUpdatedAt = NaiveDatetimeField()
     lastSummarizedAt = NaiveDatetimeField()
+    lastEmbeddedAt = NaiveDatetimeField()
 
     topics: fields.ReverseRelation["Topic"]
 
     class Meta:
         table = "RawTopic"
 
+class Embedding(Model):
+    id = fields.IntField(pk=True)
+    compressedData = fields.BinaryField(required=True)
+    createdAt = NaiveDatetimeField(auto_now_add=True)
+    updatedAt = NaiveDatetimeField(auto_now=True)
+
+    class Meta:
+        table = "Embeddings"
```

Comment:
don't we need an embedding type? I think we generate multiple embeddings for each context right?

Reply:
the compressed data is a json where the keys are the old `db_name`. in short, is a folder with the same keys we used on the `dbs` folder
<img width="401" alt="image" src="https://github.com/user-attachments/assets/31358b43-862e-4722-b843-c3306be81c6c">